packages/amplify-appsync-simulator/src/velocity/util/errors.ts

```diff
@@ -36,10 +37,12 @@ export class TemplateSentError extends Error {
 export class Unauthorized extends TemplateSentError {
   constructor(gqlMessage, info: GraphQLResolveInfo) {
     super(gqlMessage, 'Unauthorized', {}, {}, info);
+    Object.setPrototypeOf(this, Unauthorized.prototype);
```

Comment:
Q: are we using this prototype information elsewhere?

Reply:
Without setting this `instanceof` will not work